### ISSUE TITLE
can't figure out the format of print(propnet)

### ISSUE THREAD

**Issue by computron** [materialsintelligence/propnet#80]
The jupyter notebook starts with printing a propnet. I tried for awhile to figure out what the format of this thing is but it's beyond me. Why are there indents sometimes and not others? I can't figure it out!

Incidentally there is a networkx ascii tool, I'd be pretty interested to see what kind of craziness would result from printing that out ...

https://github.com/cosminbasca/asciinet

**Comment by mkhorton** [comment]
Good call, was actually looking at `tree_format` for this, which prints things quite nicely/hierarchically

I wouldn't expect asciinet to produce anything remotely legible, but might be fun to try!

**Comment by mkhorton** [comment]
@montoyjh @dmrdjenovich any thoughts on this?

Linking [tree_format](https://github.com/jml/tree-format) as previously proposed solution

Still think we need better `__str__` + `__repr__` methods, esp. for `Material` and `Graph` (née `Propnet`)
